### ISSUE TITLE
[SUGGESTION] Include basic dupes with the addon

### ISSUE THREAD

**Issue by TwistedTail** [Stooberton/ACF-3#239]
**Short Description**

One of the very first problems new players find when starting with the addon is the complete lack of knowledge on how to use it or what you can accomplish with it. I believe shipping ACF with a small selection of basic contraption could be useful to both attract more new players and also let them use them as reference so they can improve on their own.

Not only vehicles, but also useful dupe tools could be included, first example that comes to mind is the armor scanner most old players already have.

**Comment by Goodva** [comment]
good point, probably convert my acf2 pub into acf3. might as well bully karbine into doing the same with his tanks. and also a good idea for those dupes is have them use same e2's and what not.

**Comment by LengthenedGradient** [comment]
Implemented by dupe browser
